pkg/controller/apis/config/v1alpha1/doc.go

```diff
@@ -17,6 +17,12 @@ limitations under the License.
 // +k8s:deepcopy-gen=package
 // +k8s:conversion-gen=k8s.io/kubernetes/pkg/controller/apis/config
 // +k8s:conversion-gen=k8s.io/component-base/config/v1alpha1
+// +k8s:conversion-gen=k8s.io/cloud-provider/app/apis/config
```

Comment:
drop the following conversion-gen references in a follow-up, we only need the external packages:
* `k8s.io/cloud-provider/app/apis/config`
* `k8s.io/cloud-provider/service/config`
* `k8s.io/controller-manager/config`